src/Filament/Traits/HasTaxonomies.php

```diff
@@ -33,13 +33,7 @@ public function taxonomyable()
 
     public static function taxonomies()
     {
-        if (isset(static::$_taxonomies)) {
-            return static::$_taxonomies;
-        }
-
-        static::$_taxonomies = Taxonomy::whereIn('id', TaxonomyResource::where('resource_class', static::$resourceName)->pluck('taxonomy_id'))->get();
-
-        return static::$_taxonomies;
+        return Taxonomy::whereIn('id', TaxonomyResource::where('resource_class', static::$resourceName)->pluck('taxonomy_id'))->get();
```

Comment:
Why did you get ride of this storing of fetched Taxonomies?

It looks like it just needs to be changed from `isset`  to `!empty`

Reply:
Was not usable because of static